src/pages/Market/Header/index.tsx

```diff
@@ -0,0 +1,68 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import BigNumber from 'bignumber.js';
+import { Paper, Typography } from '@mui/material';
+import { useTranslation } from 'translation';
+import { formatCommaThousandsPeriodDecimal, convertCentsToDollars } from 'utilities/common';
+import { useStyles } from '../styles';
+
+interface IHeaderProps {
+  totalSupply: BigNumber;
+  totalBorrow: BigNumber;
+  availableLiquidity: BigNumber;
+  totalTreasury: BigNumber;
```

Comment:
Let's follow our pattern:
```suggestion
  totalSupplyCents: BigNumber;
  totalBorrowCents: BigNumber;
  availableLiquidityCents: BigNumber;
  totalTreasuryCents: BigNumber;
```